python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                                  'arguments as %s %r method.',
+                         'W0233': 'The abstract method %r in class %r must be overridden within a '
+                                  'concrete class.',
+                         'W0231': 'Child class should call the \'__init__\' method from it\'s base class %r.',
+                         'W0233': 'Child class should call the \'__init__\' method of the parent class '
+                                  'rather than some unrelated class.',
+                         },
+        'SpecialMethodsChecker': {
+            'E0302': 'The special method %r expects to take %s parameters, but %d %s given.',
+        },
+    },
+    'pylint.checkers.exceptions': {
+        'ExceptionsChecker': {
+            'E0701': '\'except\' blocks are analyzed top to bottom, so generic exception types should '
+                     'come after specific exception types. Otherwise, the block for the specific '
+                     'exception type will never be reached. (%s)',
+            'E0702': '\'raise\' is used to raise Exceptions. Anything else will lead to an error. '
+                     '(Raising %s)',
+            'E0704': '\'raise\' may only be called without an expression inside an \'except\' block.',
+            'E0710': '\'raise\' expects an object derived from the BaseException class. Attempting to '
+                     'raise any other object will lead to an error.',
+            'E0711': '\'NotImplemented\' is strictly used as a return value for binary special methods '
+                     '(e.g __eq__), indicating that they\'re not implemented. You mean to raise '
+                     '\'NotImplementedError\', which indicates that the abstract method must be '
+                     'implemented by the derived class.',
+            'E0712': '\'except\' expects an object derived from the BaseException class. Attempting to '
+                     'raise any other object will lead to an error.',
+            'W0702': 'It\'s bad practice to use the \'except\' keyword without passing an expression, '
+                     'since any and all expressions will be caught, which may lead to bugs.',
+            'W0703': 'It\'s bad practice to use \'except: Exception\' since any and all expressions '
+                     'will be caught, which may lead to bugs.',
+            'W0705': '\'except\' blocks are analyzed top to bottom, so if we try to catch the same '
+                     'exception multiple times, only the first \'except\' block for the exception '
+                     'will be reached.',
+            'W0706': 'If the \'except\' block uses \'raise\' as it\'s first or only operator, it '
+                     'will just raise back the exception immediately.',
+            'W0711': '\'except\' can catch multiple exceptions, but they have to be passed in as a '
+                     'tuple. Your exception to catch is a result of a binary operation (%s), '
```

Comment:
The second sentence gives too much information. Replace with an example: "For example, use except (A, B) instead of except A or B"